Mathlib/Algebra/MonoidAlgebra/Basic.lean

```diff
@@ -314,6 +314,19 @@ theorem domCongr_refl : domCongr k A (.refl G) = .refl :=
 @[to_additive (attr := simp)]
 theorem domCongr_symm (e : G ≃* H) : (domCongr k A e).symm = domCongr k A e.symm := rfl
 
+@[to_additive (attr := simp)]
```

Comment:
It looks to me like all the changes in this file can be spliced out into a PR that I would insta-merge